process-instance-migration/src/main/frontend/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "process-instance-migration-frontend",
+  "version": "0.0.0",
```

Comment:
artifact version?

Reply:
Good catch

Reply:
This would only be relevant if we wanted to publish this.

https://docs.npmjs.com/cli/v9/configuring-npm/package-json#version